doc/system_health_monitoring/system-ready-HLD.md

```diff
@@ -210,19 +257,48 @@ a new manifest variable is introduced to control whether "check_up_status" shoul
                  "<dockername>": {
                      ...
                      "state": "enabled",
-                     "check_up_status": "true"
+                     "check_up_status": "true",
+                     "irrel_for_sysready": "true"
                  }
               }
             }
 ```
 
+The feature configuration is controlled by `sysready_state` field of `DEVICE_METADATA` table.
```

Comment:
>DEVICE_METADATA [](http://example.com/codeflow?start=70&length=15)

why not use FEATURE table?

Reply:
It used. But the parameter you highlighted is used to control the global feature state. enabled or disabled. With disabled feature, it will wait for the readiness of only one component - swss "PortInitDone"

Reply:
If the FEATURE is enabled, why allow user to disable it here?

Reply:
FEATURE table controls the effect of a specific feature (sonic app extension) on sysmonitor. So it will, or won't have an effect on system startup flow.
But DEVICE_METADATA table controls the system ready state globally, with `disabled` state it won't wait for any daemon enabled in FEATURE table, but only for swss's "PortInitDone"